README.md

```diff
@@ -138,9 +134,9 @@ which the standard toast aims to accomplish natively.
   either by stacking them in the view,
   or queueing them and displaying sequentially.
 
-A broader goal of the standard toast is to provide a base for more opinionated or featureful toast libraries to layer on top of.
+A broad goal of the standard toast API is to provide a base for more opinionated or featureful toast libraries to layer on top of.
```

Comment:
Delete? To me, this just makes it sound like it might not really be a goal!

Reply:
I changed the language so it feels like more of a priority. Providing layering for libraries is important as a goal so I don't think it should be taken out, but instead emphasized differently.